src/StringParser/StringParserInterface.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Noodlehaus\StringParser;
+
+/**
+ * Config string parser interface
+ *
+ * @package    Config
+ * @author     Jesus A. Domingo <jesus.domingo@gmail.com>
```

Comment:
Please, add yourself as author of this class, removing the others.

Reply:
I've fixed this.